liquibase-core/src/main/java/liquibase/changelog/ChangeSet.java

```diff
@@ -596,7 +597,8 @@ public ExecType execute(DatabaseChangeLog databaseChangeLog, ChangeExecListener
         }
 
         long startTime = new Date().getTime();
-        Scope.getCurrentScope().addMdcValue(MdcKey.CHANGESET_OPERATION_START_TIME, new ISODateFormat().format(new Date()));
+        String startTimeString = Instant.ofEpochMilli(startTime).toString();
```

Comment:
Should we add a method getStartTime() instead of duplicating the code at 3 places?